networking_module/prepare_data.R

```diff
@@ -59,15 +58,21 @@ path.dates <- paste0('/var/lib/xroad-metrics/networking/dates', profile.suffix,
 path.membernames <- paste0('/usr/share/xroad-metrics/networking/membernames', profile.suffix, '.rds')
 xroad.descriptor <- settings$networking$"xroad-descriptor-file"
 days <- (settings$networking$interval + settings$networking$buffer)
+fetchsize <- ifelse(is.null(settings$networking$fetchsize), 100000, settings$networking$fetchsize)
 
 tryCatch(
-  con <- dbConnect(
```

Comment:
https://rpostgres.r-dbi.org/reference/postgres-query.html

Using `dbsendquery()` and than ``dbFetch()` could also help to get partial result

Reply:
I have considered several replacements for RPostgreSQL. Probably RPostgres was one of them, but I do not remember the details anymore. One problem I faced was that some libraries supported dbFetch() method, but internally were actually fetching all data into the memory and dbFetch() returned results from memory not from database service. If you wish to investigate RPostgres as an alternative please check process memory usage to make sure it does not silently fetch all the data into the memory.

Reply:
Also I did not find RPostgres in Ubuntu packages, while r-cran-rodbc can be easily installed from ubuntu repository.